### DISCUSSION TITLE
Remodify Case14 to Run Economic Dispatch

### DISCUSSION THREAD

**Discussion by JeffDecary** [ANL-CEEESA/UnitCommitment.jl#23] [Technical Discussion] [answered]
Hi,

I would like to run an Economic Dispatch given the outcome of the Unit Commitment using PowerModels.jl. I read that the instance in this package have been slightly modified. Correct me if I am wrong, but all I would need to do would be to change the mpc.gencost in the original MatPower case format, right?

Thank you,
Jeff Sylvestre-Decary

**Comment by iSoron** [comment, marked as answer]
Hi Jeff, I don't recommend making the assumption that solutions to the modified MATPOWER test cases included in UC.jl would be feasible (or even make sense) for the original cases. These modified test cases were included in UC.jl for the only purpose of benchmarking UC formulations; we did not try to keep the solutions compatible. If this compatibility is important, I recommend re-converting the original MATPOWER test cases to UC.jl format, or exporting the UC.jl test cases into a format that PowerModels.jl can read. This integration between the UC.jl and PowerModels.jl is not implemented yet, but it would be a nice contribution.

**Reply by JeffDecary** [reply]
Thank you very much!

**Reply by JeffDecary** [reply]
Hi Alinson, 

I am having a bit of trouble understanding how these instances were modified. Do you have any implementation or documentation on how it was done?

Thank you,
Jeff

**Reply by iSoron** [reply]
Hi Jeff, there is a [short description in our documentation](https://anl-ceeesa.github.io/UnitCommitment.jl/0.2/instances/#matpower). A more detailed description will be available in an upcoming paper.
